packages/browserify/examples/01-simple-js/index.html

```diff
@@ -1,7 +1,15 @@
 <!DOCTYPE HTML>
 <html lang="en">
 <head>
-    <script src="https://cdn.jsdelivr.net/npm/@lavamoat/snow/snow.js"></script>
+    <script>
+      Object.defineProperty(window, 'SCUTTLER', {
```

Comment:
just an afterthought

```suggestion
      Object.defineProperty(globalThis, 'SCUTTLER', {
```

altho same result, this might pos confuse users getting started less

since we talk about `globalThis` alot in
- original pr https://github.com/LavaMoat/LavaMoat/pull/360 where we intro `scuttleGlobalThis` (not `scuttleGlobalWindow`)
- recently https://github.com/LavaMoat/LavaMoat/pull/459
- in new docs https://github.com/LavaMoat/LavaMoat/pull/477 also mentioning original pr

then again i could be wrong, since users might expect `window` since in browser env

curious on your thoughts